packages/fx-core/src/plugins/resource/frontend/resources/errors.ts

```diff
@@ -146,10 +146,13 @@ export class StorageAccountAlreadyTakenError extends FrontendPluginError {
 }
 
 export class CreateStorageAccountError extends FrontendPluginError {
-  constructor() {
-    super(ErrorType.User, "CreateStorageAccountError", "Failed to create Azure Storage Account.", [
-      tips.checkLog,
-    ]);
+  constructor(innerErrorCode: string) {
+    super(
+      ErrorType.User,
+      "CreateStorageAccountError",
+      `Failed to create Azure Storage Account, Azure error code: ${innerErrorCode}.`,
```

Comment:
Is there any update from Chaoyi on the best practice to mask sensitive data from telemetry? If that's ready, maybe it would be helpful to output the inner exception into the telemetry.

Reply:
This issue is assigned to Changyong now. He is requesting vscode-extension-telemetry to filter out user info in error stack. Seems it still takes some time. Let's put the error code to see if it helps first.